### DISCUSSION TITLE
Total transfer progress

### DISCUSSION THREAD

**Discussion by woollarding** [airdcpp/airdcpp-windows#98] [Q&A]
I can't seem to find a way to view the overall progress of a file transfer, it only displays the current piece being downloaded. I have looked around google and even tried the web interface that I didn't know DC had until now. Any help is appreciated thank you.

**Comment by Sopor** [comment]
You mean the `Transfers` at the bottom of the window?

Have you enabled it in the menu View (CTRL+3)?

Sometimes it is only very small, try to grab it with the mouse and expand it.

You can also right click in a column and enable and disable each column.

Note: DC (Direct Connect is a protocol) doesn't have any WebUI, only the client AirDC++ and the Web Client have. 😋

**Comment by woollarding** [comment]
No. Sorry.

I see the transfers, it's in chunks.. not the whole file. Is there a way to
display it so the progress bar is the total amount of the file and not each
piece or somewhere else I can see that?
